certbot-dns-bookmyname/setup.py

```diff
@@ -0,0 +1,69 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.25.0.dev0'
```

Comment:
It looks like this should now be `0.26.0.dev0` (sorry for the delay with this review!)

Reply:
No prob. I'll update the branch against current head

Reply:
Done